dianna/__init__.py

```diff
@@ -154,4 +171,6 @@ def _get_explainer(method, kwargs, modality):
     method_kwargs = utils.get_kwargs_applicable_to_function(
         method_class.__init__, kwargs
     )
+    for key in method_kwargs.keys():
```

Comment:
Please add an inline comment here to explain why this is needed.

Reply:
Added an inline comment